forest/daemon/src/daemon.rs

```diff
@@ -583,28 +600,28 @@ mod test {
 
     use super::*;
 
-    #[tokio::test]
+    #[tokio::test(flavor = "multi_thread")]
```

Comment:
Why do we need it?

Reply:
Why is the single threaded runtime insufficient? Surely it can run multiple tasks in parallel.

Reply:
I think we need to debug why that happened. We shouldn't be doing anything blocking.

Reply:
The last time I encountered deadlock with tokio was because there were multiple runtimes. It was a bug. :)